ricks/serializers.py

```diff
@@ -14,8 +15,15 @@ class RickSerializer(serializers.ModelSerializer):
     class Meta:
         model = Rick
         fields = "__all__"
+        read_only_fields = ['user']
 
     def validate_universe(self, value):
         if self.instance and self.instance.universe != value:
             raise ValidationError({"error": "You may not change a Rick's universe!"})
         return value
+
+    def create(self, validated_data):
+        user = User.objects.create_user(username=validated_data['universe'], password='rickspassword')
```

Comment:
Era bom que a gente deixasse o criador setar a senha que quiser, não forçar uma aqui. Até por que eles ainda não tem como mudar a senha